core-services/release-controller/_releases/release-ocp-4.10-arm64.json

```diff
@@ -13,9 +13,29 @@
     "tag":{"tagRef":{"name":"4.10"}}
   },
   "verify":{
-    "arm64":{
+    "aws-arm64":{
       "optional":true,
       "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.10-ocp-e2e-aws-arm64"}
+    },
+    "serial-aws-arm64":{
```

Comment:
Assuming arm64 has to remain, this should be either aws-serial-arm64 or aws-arm64-serial, and make the rest of these names consistent.